torch/onnx/symbolic.py

```diff
@@ -1454,6 +1454,9 @@ def group_norm(g, input, num_groups, weight, bias, eps, cudnn_enabled):
 
 def _generic_rnn(g, variant, input, initial_states, all_weights, has_biases,
                  num_layers, dropout, train, bidirectional, batch_first=None, batch_sizes=None):
+    onnxActivations = ['Relu', 'Tanh', 'Sigmoid', 'Affine', 'LeakyRelu', 'ThresholdedRelu',
+                       'ScaledTanh', 'HardSigmoid', 'Elu', 'Softsign', 'Softplus']
+    variantToOnnxActivationMap = dict(zip([act_fun.lower() for act_fun in onnxActivations], onnxActivations))
```

Comment:
No need to zip
```
variantToOnnxActivationMap = { (act_fun.lower(), act_fun) for act_fun in onnxActivations  } 
```

Reply:
Without zip it creates a set, and I meant to create a dict.